### ISSUE TITLE
Error loading message ("attachmentId" for route "mail.messages.downloadAttachment" must match "[^/]++")

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/mail#827]
This came up when loading one HTML message:
```
Error loading message
Parameter "attachmentId" for route "mail.messages.downloadAttachment" must match "[^/]++" ("" given) to generate a corresponding URL.
```

Didn’t appear with any other message yet. Any idea @DeepDiver1975 @LukasReschke?

**Comment by DeepDiver1975** [comment]
on master?

This was most probably on #824

**Comment by jancborchardt** [comment]
Right, was on that branch. Do you need any more info? I can forward the mail to you if needed.

**Comment by DeepDiver1975** [comment]
> I can forward the mail to you if needed.

yes please - thx

**Comment by DeepDiver1975** [comment]
@jancborchardt please retest

**Comment by jancborchardt** [comment]
Fixed by the commit in #824
